### DISCUSSION TITLE
My GitHub inbox is always empty, I receive only Email notifications

### DISCUSSION THREAD

**Discussion by Robotatron** [community/community#42961] [New to GitHub] [answered]
For some reason I receive only email notifications when I am mentioned in a discussion or am subscribed to an issue thread.
My inbox is always empty, which should be not. What is going on?

![image](https://user-images.githubusercontent.com/95919873/210119778-1f792d36-5e8b-4426-a3f0-f588acc7a056.png)

**Comment by radar-blip** [comment, marked as answer]
You should have something in your settings under `Notifications`, like below:

![notifs](https://user-images.githubusercontent.com/39196767/210120483-cc12ba78-9cd0-4ba4-8abe-2534a9805f75.png)

**Comment by Robotatron** [comment]
@linuxlawson What a legend, thanks! Weird I had it OFF by default.
Isnt this option ON by default?

**Reply by radar-blip** [reply]
I'm not sure about it being on by default, I cant remember.
Main thing is to ✔️  the boxes you want ✔️'ed. 👍🏻

**Comment by BawdyInkSlinger** [comment]
I checked this but I still have an empty inbox. Does this only work going forward, or is it supposed to retroactively populate my inbox? I'm not sure if I'm experiencing a bug or if it's working as designed.

**Comment by saurabhm02** [comment]
bro jus Try using a different email address to see if you receive notifications.
or Try using a different email client, such as Gmail or Outlook, to see if you receive notifications.
